src/content/docs/intro/cli.mdx

```diff
@@ -17,11 +17,11 @@ The Vyuh CLI internally uses `pnpm` and `melos` to manage the packages for the
 schema and feature packages. This is to ensure that the packages are managed in
 a consistent way across all the projects.
 
-it's a good idea to have `pnpm` and `melos` installed globally. To read more
-about how to do it, refer to these links:
+For seamless functionality, ensure the following dependencies are installed and accessible in your shell path:
 
-- [Setup PNPM](https://pnpm.io/installation)
-- [Setup Melos](https://melos.invertase.dev/getting-started)
+[Setup pnpm](https://pnpm.io/installation) and [Setup Melos](https://melos.invertase.dev/getting-started) - for managing the mono-repo structure of both NPM schema packages and Dart-based feature packages. It's recommended to install these globally for consistent package management.
+
+[**Sanity CLI**](https://www.sanity.io/docs/getting-started-with-sanity-cli) - for building CMS-driven UIs and managing Sanity projects. Ensure you are logged into Sanity using sanity login.
```

Comment:
the description below was more comprehensive. Also include reference to vyuh doctor as a way of determining the environment